### PR TITLE
Implement alias action adapter

### PR DESCRIPTION
**Pull request by yongpingchen**
Same as #207. I implement adapter for AliasAction

### CODE REVIEW

File: thingif/src/main/java/com/kii/thingif/internal/gson/AliasActionAdapter.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if (jsonElement == null) return null;
+
+        JsonObject json = jsonElement.getAsJsonObject();
+
+        for (Map.Entry<String, Class<? extends Action>> entry: this.actionTypes.entrySet()) {
+            String alias = entry.getKey();
+            Class<? extends Action> actionClass = entry.getValue();
+            if (json.has(alias)) {
+                JsonObject actionJson = json.getAsJsonObject(alias);
+                Action action = new Gson().fromJson(actionJson, this.actionTypes.get(alias));
```
Comment (shinsuke-nara) [inline thingif/src/main/java/com/kii/thingif/internal/gson/AliasActionAdapter.java]: We can use  `actionClass` instead of`this.actionTypes.get(alias)`.

**Comment by shinsuke-nara** [comment]
@yongpingchen Could you check my comment?

**Comment by yongpingchen** [comment]
@shinsuke-nara  fixed. Please confirm.

**Comment by shinsuke-nara** [comment]
@yongpingchen LGTM!

**Pull request merged**